lib/web/integrations_awsoidc.go

```diff
@@ -114,11 +114,22 @@ func (h *Handler) awsOIDCDeployService(w http.ResponseWriter, r *http.Request, p
 		databaseAgentMatcherLabels[label.Name] = utils.Strings{label.Value}
 	}
 
+	var labels types.Labels
+	if len(databaseAgentMatcherLabels) > 0 {
+		labels = databaseAgentMatcherLabels
+	} else {
```

Comment:
i kept this for backwards compat just in case an old web UI defines these fields.

should i mark it for deletion or leave it as is?

Reply:
or maybe merge them?